x/stakeibc/keeper/ibc_handlers.go

```diff
@@ -159,39 +158,16 @@ func (k *Keeper) HandleSend(ctx sdk.Context, msg sdk.Msg, sequence string) error
 	coin := sendMsg.Amount[0]
 	// Pull host zone
 	hostZoneDenom := coin.Denom
-	amount := coin.Amount.Int64()
 	zone, err := k.GetHostZoneFromHostDenom(ctx, hostZoneDenom)
 	if err != nil {
 		return err
 	}
 
 	// Check to and from addresses
-	withdrawalAddress := zone.GetWithdrawalAccount().GetAddress()
 	delegationAddress := zone.GetDelegationAccount().GetAddress()
 	redemptionAddress := zone.GetRedemptionAccount().GetAddress()
 
-	// Process bank sends that reinvest user rewards
```

Comment:
I love watching `ibc_handlers` slowly evaporate

Reply:
hahaha it's beautiful